perf/tools/setup_env.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/sh
+
+# The script setting up a Linux operating system before running
+# Tarantool benchmarks, see [1].
+#
+# 1. https://github.com/tarantool/tarantool/wiki/Benchmarking
```

Comment:
You are not setting up a Linux operating system, you are setting up its kernel parameters for testing.

Reply:
The script is about setting OS environment, but yes, proposed settings are for the kernel only.